Source/Core/DolphinQt/Settings.cpp

```diff
@@ -109,6 +131,32 @@ void Settings::SetUserStylesEnabled(bool enabled)
   GetQSettings().setValue(QStringLiteral("userstyle/enabled"), enabled);
 }
 
+void Settings::GetToolTipStyle(QColor& window_color, QColor& text_color,
+                               QColor& emphasis_text_color, QColor& border_color,
+                               const QPalette& palette, const QPalette& high_contrast_palette) const
+{
+  const auto theme_window_color = palette.color(QPalette::Base);
+  const auto theme_window_hsv = theme_window_color.toHsv();
+  const auto brightness = theme_window_hsv.value();
+  const bool brightness_over_threshold = brightness > 128;
+  const QColor emphasis_text_color_1 = Qt::yellow;
+  const QColor emphasis_text_color_2 = QColor(QStringLiteral("#0090ff")); // ~light blue
+  if (Config::Get(Config::MAIN_USE_HIGH_CONTRAST_TOOLTIPS))
+  {
+    window_color = brightness_over_threshold ? QColor(72, 72, 72) : Qt::white;
+    text_color = brightness_over_threshold ? Qt ::white : Qt::black;
```

Comment:
Any reason this has a space?
```suggestion
    text_color = brightness_over_threshold ? Qt::white : Qt::black;
```

Reply:
Thanks. Fixed